CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release v1.4 (2018-07-12)
 
+## Breaking Changes
+  * Package **etcdv3** was renamed to **etcd**. This change may affects imports, configuration
```

Comment:
> may affect~s~

Perhaps just go with: "..imports, _configuration file name and flags_..".

Also I'm not sure I follow the part: "_In cn-infra.._".

Reply:
But all of those were changed: https://github.com/ligato/cn-infra/commit/d943d4cb88bd87ac610c3d907a19f6e69a1e8a4d#diff-0fb01a69bc51f27c7714cdaaf0a791d2R39